auth/service/context/context.go

```diff
@@ -41,16 +41,16 @@ func New(svc service.Service, response rest.ResponseWriter, request *rest.Reques
 }
 
 func (c *Context) Close() {
-	if c.authStoreSession != nil {
-		c.authStoreSession.Close()
-		c.authStoreSession = nil
+	if c.authsSession != nil {
+		c.authsSession.Close()
+		c.authsSession = nil
 	}
 }
 
-func (c *Context) AuthStoreSession() store.StoreSession {
-	if c.authStoreSession == nil {
-		c.authStoreSession = c.AuthStore().NewSession(c.Logger())
-		c.authStoreSession.SetAgent(c.AuthDetails())
+func (c *Context) AuthsSession() store.AuthsSession {
```

Comment:
`AuthsSession` as in Authentications  Session or Authentication Sessions?

Reply:
@jh-bate That goes away in a later commit. :)